sphinx/source/quickstart/index.rst

```diff
@@ -1,26 +1,19 @@
-Quickstart
+Start Here
 ==========
 
-First, you should :ref:`setup a CCF-compatible environment <quickstart/requirements:Environment Setup>`. Then, you will be able to :ref:`build CCF from source and run CCF test suite <quickstart/build:Building CCF>`. Note that for rapid prototyping, you can run a `virtual` build of CCF that does not require Intel SGX.
+CCF releases (``ccf.tar.gz``) are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases>`_. Once downloaded, the extracted install directory can be copied to a long-lived path, e.g. ``/opt/ccf-install``.
 
-Once this is done, you can quickly spin up a CCF network and start :ref:`issuing commands to the deployed application <users/issue_commands:Issuing Commands>`:
+The install directory contains (under ``getting_started/``) the Azure and Ansible scripts required to setup the CCF environment. See :ref:`how to setup the CCF environment here <quickstart/requirements:Azure Confidential Compute>`.
 
-.. code-block:: bash
+.. note::
```

Comment:
There is no easy way to test that an install actually works out-of-the-box. I've raised https://github.com/microsoft/CCF/issues/1028.

Reply:
I think the docker image with an authless logging app we talked about yesterday is what we want. We can include both an SGX and a virtual build on there.

We do test the install in our release pipeline, I think that should be sufficient.